pkg/ccl/testccl/sqlccl/sqlccl_test: TestExplainGist failed

pkg/ccl/testccl/sqlccl/sqlccl_test.TestExplainGist [failed](https://mesolite.cluster.engflow.com/invocations/default/073d610e-ab40-4bc1-a7ac-4bcd741d16b9?testReportRun=2&testReportShard=2&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvc3FsY2NsOnNxbGNjbF90ZXN0) on master @ [fd8129a1c152c527c43c94d5cb8355ab74e9dd4d](https://github.com/cockroachdb/cockroach/commits/fd8129a1c152c527c43c94d5cb8355ab74e9dd4d):


```
        	GOROOT/src/database/sql/sql.go:1672 +0x4f
        database/sql.(*DB).retry(0x0?, 0xc007405f10)
        	GOROOT/src/database/sql/sql.go:1576 +0x42
        database/sql.(*DB).ExecContext(0xc007da9230?, {0x8aad870?, 0xdc40c80?}, {0xc00a376000?, 0x0?}, {0x0?, 0x0?, 0xc00600dfd0?})
        	GOROOT/src/database/sql/sql.go:1671 +0xc8
        database/sql.(*DB).Exec(...)
        	GOROOT/src/database/sql/sql.go:1685
        pkg/ccl/testccl/sqlccl/sqlccl_test.TestExplainGist.func1.6()
        	pkg/ccl/testccl/sqlccl/explain_test.go:297 +0x46
        created by pkg/ccl/testccl/sqlccl/sqlccl_test.TestExplainGist.func1 in goroutine 33
        	pkg/ccl/testccl/sqlccl/explain_test.go:296 +0xc9e
        
        goroutine 9942 [select, "job":"AUTO UPDATE SQL ACTIVITY id=103", "n":"1"]:
        github.com/cockroachdb/cockroach/pkg/sql.(*sqlActivityUpdateJob).Resume(0xc00b1246f0, {0x8aadaf8, 0xc010f087b0}, {0x7632ca0, 0xc00b124910})
        	pkg/sql/sql_activity_update_job.go:108 +0x2b4
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2(0xc001f5ea80?, 0x0?, 0xc005de78e0, {0x8a86120?, 0xc00b1246f0?}, {0x8aadaf8?, 0xc010f087b0?}, {0x7632ca0?, 0xc00b124910?})
        	pkg/jobs/registry.go:1653 +0xb4
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine(0xc0062c1008, {0x8aadac0, 0xc010f06460}, {0x7632ca0, 0xc00b124910}, {0x8a86120, 0xc00b1246f0}, 0xc001f5ea80, {0xc0108b35b2, 0x7}, ...)
        	pkg/jobs/registry.go:1654 +0xb7f
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob(0xc0062c1008, {0x8aadac0, 0xc010f06460}, {0x8a86120, 0xc00b1246f0}, 0xc001f5ea80, {0xc0108b35b2, 0x7}, {0xc010c274b8, 0x7})
        	pkg/jobs/adopt.go:425 +0x4f9
        github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func1({0x8aadaf8?, 0xc010c4db90?})
        	pkg/jobs/adopt.go:278 +0x13c
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func1({0x8aadaf8, 0xc010c4db90})
        	pkg/util/stop/stopper.go:440 +0x8a
        created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 9912
        	pkg/util/stop/stopper.go:438 +0x125
    explain_test.go:312: stmt wasn't canceled by statement_timeout of 0.1s - ran at least for 1m: DO $$
        DECLARE
        BEGIN
        WITH with_271 (col_2132, col_2133) AS (SELECT * FROM (VALUES ('''FDNfRXXb'':67C,480C,734C,925C ''FidAAlIaJ'' ''VtyP'':522C,713B ''clN'':58C,355B,934C ''deDJs'' ''gFfXHDdZ'' ''lIO'':371B,702C,806C,990A ''sNfHIzKHoL'' ''vHrNAODq''', e'\U000282C6W\U00016A97\U00022C94\U000255C7' COLLATE en_US), ('''BTIwxeNinF'':73A,376A,749C,915C ''FQJ'' ''HBgC'':706B,715C,918B ''aStqIhyk'' ''dnNC'' ''dsDo'' ''qWsu''', e'\U0002CE68\U00005F3A' COLLATE en_US), ('''rVMkuH''', e'\U00026F3A\U000043C7\U00009E1F\U000203C1\U00009E36' COLLATE en_US), ('''CcsfqUAj'' ''IUhx'':564A ''PKwGwuDB'':691C ''dKThbuFRb'':201C,599A,709A,775B,814A ''h'':390B,622A ''ip'':79A,103A,463B,543C ''lhpETt'' ''yE'' ''yJxrNkdx''', postgis_scripts_released()::STRING COLLATE en_US), ('''AWLsIXBlnH'':312C ''DOjNvWO'' ''LRgUtGp'':146C,705B ''VNOlJdW'' ''XHnHUzV'':43C,437C ''XaK'':96A,632C,770C ''ZvTYPnq'':664C,970C,974A ''a'':319B,976A ''sxoC''', '' COLLATE en_US), ('''JbFqLXCm'':447B,743A', e'\U00028038' COLLATE en_US)) AS tab_1323 (col_2132, col_2133) EXCEPT ALL SELECT * FROM (VALUES ('''BIJqiutK'':474B,782C ''QAALXjxPi'':33C,129B,501A,728C ''ghSNgx'':450B,936C,947B', e'\U0001680B\U0002E37D\U00029C95\U0002CEB9\U00030FA7\U0000437B\U0002C442' COLLATE en_US), ('''N'':188C,980B ''fbauWXa'':445C,881B ''zmhH''', e'\u0402\U00007F0F\U0001057C\U00028EFC\U00031706\U0002B859' COLLATE en_US), ('''AbFbFnjqpI'' ''EjlHgm'' ''N'' ''QH'':101C,855A,971B ''UNExqWd'':362A,708B,779B,983B ''UlqXl'':313C ''cPgx'':13C,742C,995C ''fBXwT'' ''yPGyfUJT''', e'\U0002519B\U00028115\U00017989' COLLATE en_US), ('''LNI'' ''QRMIalA'' ''e''', NULL), ('''ypNFq''', e'\U0002454C\U00028826\U000124EE\U00007294\U00017BC7\U000063EF\U00011003\U00003D2F\U00023C53' COLLATE en_US), ((WITH with_270 (col_2134) AS (SELECT * FROM (VALUES ('[-1.2073008,0.8033924,0.68243325,1.4949651,-1.2751727,-1.816943,-0.34623262,-0.2563357,-0.42412493,0.031534515,0.15706114,-0.39988866,-1.7067055,1.2417061,2.4600008,-2.0146885,1.0243449,-0.16872346,0.1919279,0.27057523,0.48632658,-0.052021164,-1.7022574,-0.97762823,-0.37474257,0.23008384,-0.967267,-0.08221417,-0.34715593,-1.0329709,-0.35983488,0.42757553,1.1943258,0.44556662,1.4751494,0.07182167,-1.0204431,-0.34257945,-0.5328388,0.3182807,-0.2734197,-0.8277642,-2.1363585,0.2567086,-0.36904275,-0.59176177,0.10789969,2.8861094,1.5174133,0.22277091,1.0019879,0.94419515,1.349572,-0.9577438,0.56218356,-1.4284781,1.0240284,0.43831316,0.025357034,-0.3614549,-0.14435418,0.056777902,1.3328968,0.60906357,-0.13156624,-1.3529909,2.6378038,-1.6099936,0.80696476,1.2473893,-1.2689868,1.416102,-0.26260164,0.037148982,1.0516987,1.2243247,1.7797484,-0.29345915,0.9444958,2.997505,0.5489504,1.0659382,1.2755483,0.8961934,-0.60070634,0.03244464,-0.91375613,-0.74855125,-0.35388,0.29257375,-0.92288476,-0.4022346,0.1864421,0.3133529,0.9325749,1.5482124,0.60406953,-1.2395916,-1.0843272,-2.131103,0.042910445,0.8356566,-0.41749108,1.6264095,0.47368327,0.32228193,0.80233103,-2.1326275,-0.36427918,0.9833828,0.99907094,0.4318368,1.5652944,-3.694743,-0.59274125,0.44625717,-0.30916154,1.506698,-0.407863,1.7588563,-0.18206932,-0.29404113,-0.50885904,-0.32761714,0.3748712,-0.890944,-0.20964971,-0.095715225,-2.005549,-0.29250628,0.02628203,-1.0879092,0.8990509,0.420008,-0.3613947,-1.4400403,-0.8863732,0.5257203,-0.78557694,0.03014159,2.1740897,-0.012730218,0.847856,1.023314,-0.2566902,0.42799467,-0.22984101,1.0005945,-0.44154194,-1.7190411,-0.28318807,0.8420632,0.89264804,-0.9344192,-0.64493525,-0.53505486,0.30911085,1.0934207,0.7400766,-0.6176055,0.258869,0.35125944,-1.6049387,-1.7257946,-0.47111204,0.5511965,1.3375036,-0.5195344,-0.025024682,-1.1794341,-1.1372018,0.83976215,-1.0892708,1.3242753,0.1343089,-0.5443474,1.8526659,-0.20783842,0.64703524,-0.2760632,-1.0979818,-0.8705943,0.43279657,-0.19492307,-0.9654204,0.23166214,0.54019517,1.3954918,0.68890065,0.573876,-0.3984189,1.1640613,-0.652065,0.12102975,-1.7489878,-1.3113453,-0.36371502,-1.0980555,0.8025999,1.4902911,0.86302555,-0.62097263,-1.0227779,-0.19077253,0.7904418,-0.35057884,-0.9657898,0.24458031,0.36748442,-0.68556094,0.94292915,-0.79503095,-0.24623185,-0.007760113,-1.6960579,-2.5401988,-1.0243249,1.1036159,-1.0644227,0.17530954,3.1886196,-0.42376617,-0.53358054,-0.44521782,0.58390284,-0.9961116,-2.2937133,0.68239594,-0.4380695,-1.6042649,1.7032547,-0.43074828,-0.42197067,-2.4774683,-0.99207497,0.014551462,0.3724993,1.2869985,0.48124984,-0.2143193,0.07115101,-1.1504493,-0.83857137,0.59294623,-0.09970369,-0.15572146,-0.57921493,0.18101092,-1.0398734,-0.65739274,0.01019096,1.1011099,-0.8856989,0.45180973,-0.3285733,2.0067222,0.3874655,0.9679709,0.24448161,0.306941,0.3067343,-0.57247984,0.6235955,0.0837329,1.8687105,-0.6030016,0.8164124,0.66689795,-1.9506145,-0.39595112,-0.2407789,-1.2897273,0.36809728,0.199131,-0.017010648,-1.1303087,-1.7945796,1.1884929,0.11422041,0.9855328,-1.1552348,-0.4890443,0.046402838,-1.2751224,1.152529,-1.0491197,-0.8708562,0.2762117,-0.21382506,0.44861516,-0.058231123,0.4690563,0.5774736,-0.25991482,-1.2095859,-1.098865,0.5252288,-0.62875605,-0.23993485,-0.13099599,1.1393178,0.33679485,0.51355565,0.040284835,1.0724627,0.7095992,0.15907858,0.5398992,-1.1297762,0.64966583,1.6790684,1.0582594,0.30690917,1.4162437,0.30008578,-0.38660926,0.32345644,0.1234397,-0.88202184,0.24147871,0.8610958,0.8698965,0.51505464]'), ('[-1.4215745,-0.953495,1.0037218,-0.39741084,0.8680439,0.07328193,1.4165391,0.51763475,-0.47002742,0.65657496,1.6224501,-0.97223735,-0.48166522,0.4288805,-0.069388516,1.1893654,-1.0046171,0.6863499,-0.67495555,0.4467499,-0.09041026,-0.21924655,0.22549872,-0.7892158,-0.5607972,-0.6972738,-1.9269341,-0.767748,0.4784803,-0.79118687,-2.1931674,0.49265182,0.13752346,1.1246587,-0.74150234,0.23804577,0.32795238,1.3152585,-1.78944,-0.71707886,-0.2976329,1.2490942,-0.62851024,0.27982634,-3.523393,-0.8740006,0.5437853,-0.0700989,-0.61884314,0.23502855,0.025430711,0.41580278,0.7917379,-0.9096753,-0.089387804,-0.13344094,-1.0312133,0.89839476,-0.7363775,-1.591696,0.9991922,-0.18547417,-1.4536502,-0.68432766,0.5353801,1.048484,-0.4673963,-0.623349,-0.5441936,-0.31059867,0.030253226,-0.6681427,0.29167807,0.13601977,-1.0436127,-1.3267181,0.20723994,1.600551,0.5549024,-2.577729,0.9617922,0.15619756,-1.5777231,0.7561376,1.8594382,-0.1270812,0.9434175,-0.19598219,0.29834008,-0.3821375,0.17764254,1.5133979,0.64005595,0.722461,-0.75651795,0.045157768,0.13006042,1.2449377,0.4464879,0.84818274,-0.38948017,0.98270345,0.54431665,-0.07036009,1.4197097,0.08053927,0.42462808,0.89008266,0.14848445,0.29309425,-0.014286372,0.8655297,0.2785319,0.7409628,-0.70684147,0.00021818762,0.6308388,0.4057208,-0.23181558,-1.3666068,0.15951844,-2.7280552,-0.7251704,1.66893,-1.7777187,0.42888427,-1.3010099,0.012004538,-0.16654882,1.5462747,-0.35548142,-1.5284,-1.0614425,0.063517354,0.43545577,0.14941202,-0.34070954,-1.1985929,0.9166264,1.2319716,0.96759397,-0.95271975,-0.46702236,0.35917497,0.6068567,0.32769012,-1.2023486,0.0023675805,-2.4405532,-0.016687268,1.0330856,0.8063399,-2.8614602,-2.783688,-0.01672798,0.5209141,-0.004190738,-0.8685649,-1.3761926,-0.282698,0.15942277,-0.12854645,0.07525475,0.15094686,-0.5457892,0.99867,-0.033676364,0.7835686,-0.42313573,0.023706371,0.029918797,-1.7905536,-0.5939486,0.2486245,-0.43268377,-0.32200179,0.390544,-0.6288129,-1.466108,1.9044436,0.41324633,0.93875116,0.587343,0.42555797,-0.3169379,1.3033178,-0.30675176,1.1957321,0.64517355,0.34999776,-0.91788054,0.060886063,-0.2989289,0.22060688,-0.21294785,-0.541026,-0.180607,-0.052350085,0.8906375,1.2053406,-0.6508866,-1.3333149,0.49037763,0.7737837,0.34267813,-1.4609549,0.26333147,1.0250623,0.26361328,0.430864,0.46949953,1.1771245,-2.19182,1.6186303,-1.0665046,1.7538621,1.9410009,0.5811338,-1.9178102,0.3145629,0.10925043,-0.10291499,0.74281234,1.0605252,0.33713385,-0.67020845,-1.8070127,0.07620434,0.993488,1.3611988,-0.70411915,-0.89453965,-0.10122651,-0.8699103,-0.2280278,-0.037016395,0.79005605,1.315147,-1.2692295,0.036268722,1.2690787,-1.3907853,0.5697644,0.85619044,2.2571845,0.012754977,1.0658116,-1.0574135,-0.96192473,-0.20372632,-0.59799296,0.09101894,0.5964107,1.4985054,1.3988342,-0.23688024,-0.25866434,0.6129079,1.5262223,-1.0220383,-0.25247598,1.1419207,0.4919929,0.3951534,0.11607127,-0.45035633,0.15845498,0.15136258,1.4090397,0.15396419,1.0848176,-0.55610895,-0.1265976,0.30154562,0.28076473,0.09341423,0.52912,-1.7014337,-0.7002951,0.67968506,0.62987715,0.010713915,-0.7847566,0.014236637,-0.69980603,-1.0821967,-1.4921921,1.6803242,1.5159315,-0.35831317,-1.8804893,-0.3844948,-0.38211516,0.8596469,-0.96951216,-0.16741754,0.04105914,0.0849417,0.50179386,-0.92865354,0.12071264,0.30536363,1.2294546,0.1639239,0.0071887844,0.7266932,1.054789,-1.2119435,0.83475274,-0.8968852,0.37760952,-1.2716732,-1.29427,0.3976245,1.348014,1.6159837,-0.564118,-0.251342,-0.08667671,-1.0358855,-0.64771754,0.80731314,-0.20973428,-0.9021611,0.73558414,-0.3524596,0.0977928,0.18363136,0.025070466,0.6998045,0.49476933,0.189544,0.8672398,-1.1788723,0.94187945,0.90384203,-1.4539404]'), (NULL), ('[-0.80360574,-0.73693633,-0.36507076,0.31426564,-0.048773073,1.9729077,0.6470253,-0.796654,0.2071601,-0.71717787,-1.9448427,-2.4127946,-0.78256506,0.18114386,1.3772224,0.6557951,-0.025918221,-1.513502,-0.9247647,1.1726991,1.0763954,-0.6518571,-1.0254208,-0.83041584,-0.065781265,-0.109701864,1.2013525,-0.45625788,-0.3010704,0.08880098,-1.405488,0.66776806,-0.4352816,-1.0444394,-0.17358479,0.04889288,0.54107726,-0.73905426,-2.1263316,0.1776984,0.9936638,0.21664335,0.20886198,-1.2347951,-1.076079,0.7021908,-1.2375247,-0.99113667,0.8184358,1.7800564,0.6933168,-0.35453397,0.1011917,0.73143846,-0.6598127,-0.59468454,-0.4992604,0.18032865,-1.5600227,-0.0990518,-0.6294788,1.1101402,-0.20438951,-0.04566614,0.14332348,0.9516368,0.44733724,-0.18810262,0.3626157,1.5711399,-0.42867407,-0.93451655,-1.8911355,0.21357593,0.06959526,0.4606659,0.66188467,-1.2640663,0.35096657,1.0437633,-0.465638,0.18834789,0.7739663,-0.38572323,0.7793686,0.17616268,-0.17910269,0.28365973,-1.1692148,0.21948767,-1.0913272,-0.20258372,-0.06750029,2.055651,-0.85867125,1.1811979,-1.2328147,-1.944029,-0.44677234,1.7534701,-0.46427274,-1.0628481,1.5723996,-0.9971606,-0.33323947,-1.6257657,1.9419026,-0.8548941,-0.4109976,-0.9096835,-0.36314094,0.9168806,0.87346643,1.3395752,1.5565708,-0.44082454,-0.36502603,1.501053,0.24368674,-0.13819608,-0.44473943,-0.2804404,0.08099592,-0.06469395,0.0023969333,1.4061223,0.25902426,-0.21529518,-1.4961907,0.83883286,-1.022378,-0.6214015,0.99995434,4.316843,-0.39002132,-0.5260215,-0.1488291,0.33057114,0.23594569,-0.6769204,1.6780612,0.6755359,-0.11633653,-1.3379686,0.6299841,-0.62420034,-0.4608576,0.21268716,-0.13357872,1.9456711,0.6704658,-0.42693803,-0.3767744,-2.4482625,1.7201353,-1.1912602,-0.747059,-0.4311563,-1.8950373,1.1090957,-0.5529564,0.93726456,0.015410002,-1.45798,1.4931703,-0.9701747,-0.7701527,0.30443752,0.2752104,-1.8912033,-1.2261151,-1.0543007,-0.054378506,-1.1832513,1.9749991,-2.075612,-0.23753262,-0.56141514,-0.52573,0.47349498,-1.4088962,-0.10637274,-0.9845784,-1.869027,0.38356256,0.7190381,-0.06423549,0.37179324,0.56267285,0.5439751,0.8660878,-0.7746326,-0.09642151,-0.9391114,-1.4355257,-0.7286425,0.27181575,-0.9480423,-0.11241107,-0.35973313,-0.20950879,-0.8350906,-0.62339425,-0.28079474,1.1317704,-3.924886,-0.7037263,0.3497889,-0.31877396,-0.4365706,0.029297978,-1.2792559,0.7807533,-1.1605856,0.2673361,-0.81027657,-0.501745,0.4718334,1.0416775,0.6818244,0.8298779,2.0149846,-0.7217602,-0.13170911,-0.74335164,-0.010355775,-1.3662306,-1.0245572,0.044656083,-1.1419678,0.38429284,-0.97107667,-0.48957705,1.806384,-0.2070765,1.8497632,-2.0286658,-0.6464798,-0.51838124,2.539114,1.0733666,-0.45808947,0.73259556,-1.4506553,-1.5509492,0.3004706,-0.39879632,0.67070246,-1.465135,0.5855556,-1.3874061,-0.31136373,0.7178072,-0.8053645,1.2910098,-0.81494623,-0.61614156,-0.5164936,1.3006151,1.0017195,-2.433594,1.0914409,-1.0864276,0.21950284,-1.1802888,-0.9536241,1.0567921,0.35360596,0.38831684,-0.2549464,-0.2503218,0.7405251,0.85593414,0.2590054,0.25898924,1.9187686,-0.79126835,1.0752298,0.6100991,0.31620115,-2.4075878,-0.45796657,1.2048045,-0.30880022,0.11701299,0.5518125,3.187758,0.58310896,0.006879207,0.38216043,0.77008855,1.1513649,-1.4043851,0.326466,1.2234669,0.092509925,1.2419441,0.43483922,-1.1826435,0.10144971,2.5313077,-0.9686553,0.14672521,-1.905632,-0.9989525,-0.40496394,0.75009406,0.3321284,-1.2583519,0.33800647,0.4887442,-0.49929717,-1.5270925,0.13575256,-1.3905905,-0.48138472,1.2334955,0.7429316,0.8364906,0.11751629,-0.25631484,-1.4632723,-0.5808739,1.7458073,-0.0033251285,1.8904918,0.45251223,0.41796753,-1.0985092,0.34335768,0.2157045,-0.29744902,-0.97367775,0.027205927,0.2805671,-0.3277182,-0.43294084,2.0739374,0.6182939,0.027599761,1.3515842,-1.5447398,0.21464777,-0.72329104,0.4746189,-0.25504613,-2.0075672,-0.53456557,0.40736398,-1.4495037,0.19547522,1.097567,-0.037474334,0.277835,-1.2109838,1.3777418,-1.628907,1.5606437,0.26891077,-1.1168149,0.85081047,1.9981221,1.1540244,-0.49759877,2.6673167,1.0009371,0.10341458,-0.7154929,1.6951836,2.670601,-0.40833524,-1.7428861,1.0221612,0.7962933,0.10442832,0.3700117,0.2846669,0.71288323,-0.42774096,-0.6455935,-0.022261368,1.9826431,-0.55629086,2.001519,0.652218,-0.13888861,-0.7742137,-1.343305,-0.44814193,-0.765379,-0.38959798,0.548083,0.6130169,-0.8530584,-1.0520916,0.08375673,-0.35901415,-0.77354676,-0.7451218,0.3334107,2.003001,-0.7375398,-0.2136552,0.18120694,0.953991,1.2353132,-0.4550963,-1.683803,1.6725497,-0.90805256,-0.4083631,-0.3493695,0.52094775,-0.6508154,0.7845335,0.41632986,0.1986373,2.3416295,-0.724382,1.1026582,-1.5513496,-0.95586807,0.24916601,0.8741062,-1.0873922,2.592202,-0.6303126,-0.4079224,-1.5546921,-0.9350373,-0.21219261,-1.464144,0.6618823,0.0098276315,-0.25818488,-0.26940003,1.5757672,-0.68151486,-0.1791911,1.2131395,-2.1499732,0.29994404,0.39231798,-0.040676944,-0.008559707,-0.782275,0.27604216,0.3916516,0.49769247,-0.89514285,-0.51363647,0.08713543,-0.2627225,1.3371836,-0.37096116,-1.6808082,0.39232367,-0.20192944,-0.76702744,-0.5448362,1.8544338,-0.27788043,0.0070875115,-0.013116519,0.74416625,-1.1519723,0.6466653,1.5081904,0.90273213,1.0185086,0.46723238,-0.9856748,0.2839284,1.1302435,-0.1810346,0.918001,-0.80822617,-1.1928455,-0.27937368,0.77483493,0.29337212,-0.46869782,0.73164624,1.4301631,1.4141766,1.9466572,-2.4222674,0.77904344,0.66924554,-0.52439225,-0.33508283,0.81726885,-0.8586801,0.056000736,0.9634567,0.36434737,-1.2510529,-0.25352338,-0.42155346,1.6465044,-0.23015204,0.1066184,-0.9252011,-0.1008552,-0.98673385,0.9713898,0.3821182,-0.20166764,-0.82715493,0.38679853,0.90972,-1.3094249,1.6915979,0.98850197,2.0979974,0.9199182,-0.7416983,-0.3766644,0.618228,0.01279862,-0.6023237,-0.10242944,-0.91310954,0.9251759,1.9560676,-0.6973128,-0.64903945,-0.50285643,-0.010255121,0.926949,-0.0012644992,0.520901,-0.7492014,0.7389732,-2.070232,-0.7486903,-0.7847386,-1.6516199,0.6928246,0.55563587,-0.16705717,-0.64515704,-0.05931613,-0.8515395,-0.1221904,-0.23865986,-1.2377846,-2.1051085,-0.52263474,0.8532357,0.16688916,-0.009678184,0.656678,-0.47766462,0.6469349,0.7156824,0.61546034,-0.20707718,1.9306511,0.38793617,1.4290076,-1.2975494,-0.9112847,0.8272248,1.2667341,0.31261665,-0.37516826,1.43904,0.5815765,-0.6945581,-1.5397816,-1.2574434,0.55823183,-1.7101616,0.8168007,1.1626787,0.82847506,1.1761822,1.0776703,-0.23181134,-0.66057014,-0.6917629,-0.49909145,0.23952584,-2.129481,1.254441,-0.35236332,-2.5255117,0.25340232,-0.48977426,1.5260918,0.22436202,-0.51521224,0.27073905,1.8921474,-1.4364439,1.9496347,1.0190759,-0.5107734,1.0772468,-0.019715538,1.5390792,-0.083795145,1.3025923,-0.49658212,-1.2248341,-0.15512599,0.44803342,-2.216179,0.94215953,-2.3606856,1.5171816,1.0580862,-1.3620085,-1.17953,1.3814106,0.89629656,-0.25178745,0.005385903,1.246069,-1.08366,-0.6642963,-0.72293025,-0.62698793,1.2410315,1.7730314,-1.0365015,-0.35331327,-2.2899349,1.4608216,-0.4003107,1.9585236,-0.831611,0.4498387,0.1461168,1.0070245,0.46667394,-1.1220057,-0.8359576,0.40352193,1.6311252,1.1304849,-0.7535365,0.4645089,-0.47952247,-0.029022975,-0.05069756,-0.20900594,-0.99569696,-0.21447197,0.09637733,0.046879712,1.2148964,0.62469953,-0.96459067,0.70569247,-0.6033009,-1.176081,-1.217125,1.1692684,-0.6622197,0.13299236,-0.5545238,-0.23552248,0.6757351,1.1660949,-0.5689687,1.3831457,0.5937084,-0.44599897,-0.42907393,0.46078315,-0.3407632,-0.21689959,-0.19825369,-1.3303574,1.8277677,1.0390573,0.72214824,-2.4401305,1.7411175,-1.0976837,-0.31960738,-0.2747963,0.43617377,-1.4020288,0.16844177,-0.7524065,-0.14078248,1.4102685,1.6401066,0.113195084,-0.49578914,-0.38493758,-0.38133755,-0.2591529,-0.5849372,0.3127493,-0.08524987,-0.026833436,1.1634092,0.76093024,0.12927932,1.0739416,-0.89367247,0.41360068,0.55112827,0.3511409,0.77100146,-0.9785023,0.5764462,-0.1704289,0.85436743,1.1188267,-1.2174451,-0.72357684,-0.46008515,-1.8429441,-0.2539786,1.0835274,-0.6530685,0.37387127,1.0036781,0.3302438,0.81483805,-0.4257349,1.097226,-0.5560262,-1.3036286,-0.7412263,0.15337867,-0.0011382146,1.4145614,1.2010487,0.0741264,0.089090645,1.4110801,1.6732603,-0.29527292,-2.0733206,-0.28049967,0.8033008,-0.18658248,-1.242364,-0.54830736,0.16627699,1.2776127,0.43563136,-0.722795,-0.5269813,-1.2077264,1.2300835,0.1546079,-0.14466342,-0.42722872,0.9300272,0.016033826,0.50767523,0.02747493,1.2748146,0.8751969,-0.3818904,0.11376254,-0.65862286,-1.1370753,0.70148236,-0.8081004,2.6743903,1.0826999,0.93142706,-0.4220036,-2.924942,0.21160784,0.23457561,-0.06127068,0.65733033,1.2513455,0.20849946,-0.12551747,0.56601477,-1.6695582,-1.3589507,-0.20137581,0.38356593,-1.8132311,0.5309603,-1.5348208,-0.44994125,-1.9092113,-1.4894305,-0.119482994,2.1469946,1.2754823,-0.18542051,-0.36241603,0.17673746,-1.8300666,0.972088,0.6970188,1.6237314,-0.8068475,-0.2721816,2.1391144,-0.18301421,1.3726591,0.54101175,-0.34487405,-1.3183751,-0.53279275,-1.4582746,-1.04428,-0.2541871,-0.6302864,-1.9701773,-0.42501438,-0.2714535,0.7680395,0.22788401,-0.007299202,1.204383,0.026281597,-0.92999125,-0.47852233,0.12172334,-0.36388615,0.78454494,1.3726614,-0.15626094,-0.50282663,-0.46442303,1.6104975,0.7775233,-0.38351703,-1.4042915,-1.4339564,0.515063,-1.1797074,-0.18940979,0.19255354,2.6036465,2.305335,-2.3511224,-0.24719845,0.75519264,-1.0230546,-1.2155704,1.5471561,-0.3911808,-1.2568917,0.8870831,1.1935214,-0.88888884,-0.74861664,-0.5501316,-0.9580027,-0.20985465,0.98860544,-1.1595231,-0.6563411,1.2143244,0.7521359,-0.013820938,-1.7524085,-0.8214927,-0.4483614,0.25793076,0.0315943,1.7935069,0.24613398,0.45333982,0.462828,-0.057122596,0.133789,-0.39501113,0.16544728,0.28017387,-1.4314944,0.46499512,-1.5374634,-0.74954206,-0.77625144,0.73463064,-0.109730735,-2.0862272,1.8089501,1.3740475,-0.17055532,1.0801266,-0.26966843,-0.039053563,1.1712229,-1.10514,-0.36755016,-0.9428959,1.7356932,-1.7322167,1.5052112,-0.11849335,-1.1079161,-0.5555355,-0.48929954,-0.63503724,0.45796707,-0.6569129,-0.7961347,-1.1982453,0.09602703,0.43270957,-0.5637701,0.61507386,-0.75210327,-1.0284328,-0.4786727,0.112794496,1.1711904,0.2247025,-1.1075146,-0.8380309,0.38326845,1.3241415,0.11516353,0.34597406,-0.5285179,-0.3446732,0.41444704,-0.74845266,-1.0148017,-0.22238737,-0.0059979027]')) AS tab_1324 (col_2134)) SELECT '''BnScpKeDf'' ''JoysJKLfE''' AS col_2135 FROM with_270 AS cte_ref_89, defaultdb.public.seed AS tab_1325 GROUP BY tab_1325._date, tab_1325.tableoid LIMIT 1), (('''f'' <-> ''OoRtEadhNf'' | ''O''',) || (('{}'::JSONB - -9223372036854775807::INT8)::JSONB::JSONB->>-353207830443325513::INT8)::STRING::STRING)::STRING COLLATE en_US)) AS tab_1326 (col_2136, col_2137) EXCEPT SELECT * FROM (VALUES ('''JHTA'' ''Z'' ''glMYtfC'' ''hV'' ''iuuYCi'':520B ''kX'' ''lPmlXjcq'' ''oSbLDZviEK'' ''xT'':357A,484C,590A,743C,909A ''yRwDzaG''', e'\U0000B244\U0002CED5\U0001F59E\U0000F941\U00003A55\U0000C558\u07E1\U0003048B' COLLATE en_US), ('''Aepps'' ''VKGhY'':175B,265B,308C,525C ''qIF'' ''tPxJYvs'' ''wCa'':543C,746A,811A,875C', e'\U00008A48\U000281C8\U0000650E' COLLATE en_US), ('''IrD'':231A,465C,473A,978B ''Xn'' ''otHNeCP'':375B,392A,437B,720B', e'\U0002B55B\U000037CA\U0000C9EF\U00030BE1\U00026540\U000312D4\U0003117E\U00022C0E\U00004565' COLLATE en_US), ('''D'' ''aLLk'' ''gHKMwn'':233C ''nOYtpZB'' ''ngrkNK'' ''tRORxdx'':22C,427B', e'\U0001D82F\U00001F02\U00012441\U00000E4C\U00030F19\U00022A1A\U0000A2C7\U0002D20D\U000184D4' COLLATE en_US)) AS tab_1327 (col_2138, col_2139)) SELECT tab_1328.crdb_internal_origin_id AS col_2140, tab_1328._bytes AS col_2141, tab_1328._float8 AS col_2142 FROM defaultdb.public.seed AS tab_1328 WHERE false;
        NULL;
        SELECT '5fa001c1-ad6e-4571-83ea-8e946cf52e8b' AS col_2143, '2004-05-28 12:34:52.000063' AS col_2144, tab_1329._bool AS col_2145 FROM defaultdb.public.seed AS tab_1329 LIMIT 37;
        WHILE true LOOP
        IF true THEN
        	WITH with_272 (col_2146) AS (SELECT * FROM (VALUES (1257309107), (945951919), (1078495041)) AS tab_1330 (col_2146)), with_273 (col_2147) AS (SELECT * FROM (VALUES (e'\x068zDR%%\x0b'), ('X'), (e'%^\tDc'), ('i'), (e'@=3v\x7fBd"1')) AS tab_1331 (col_2147)), with_274 (col_2148) AS (SELECT * FROM (VALUES ('153.75.93.181/4')) AS tab_1332 (col_2148)) SELECT NULL AS col_2149, tab_1333._enum AS col_2150, tab_1333._inet AS col_2151, 3040303947002758139 AS col_2152, false AS col_2153, -2.799551486968994 AS col_2154, tab_1333._bytes AS col_2155, tab_1333._jsonb AS col_2156, tab_1333._timestamp AS col_2157, '0102000020E610000006000000C66FCB74B1DE54C08CD8D8F3715B30C010B69A99501438C0AC2E13AE629537C0FCAED9DDBD16454046B373A302ED51C0C8C8E478A2C74540E8B20FD7A0AE36C008C3F66026785A40A4A69266838138C09463858F99C64A40C80286D057262840' AS col_2158, tab_1333._float4 AS col_2159, e'g\x1d\x04n]\x114<' AS col_2160, cte_ref_90.col_2147 AS col_2161, tab_1333._uuid AS col_2162, tab_1333._int8 AS col_2163, tab_1333.crdb_internal_mvcc_timestamp AS col_2164 FROM with_273 AS cte_ref_90, defaultdb.public.seed AS tab_1333 WHERE (true AND tab_1333._bool) ORDER BY tab_1333.crdb_internal_origin_timestamp, tab_1333.crdb_internal_origin_id DESC NULLS FIRST;
        	WITH with_275 (col_2165, col_2166) AS (SELECT * FROM (VALUES ('10:49:05.393243+08:59', ARRAY[2976330034])) AS tab_1334 (col_2165, col_2166)) SELECT '1974-06-18' AS col_2167 FROM defaultdb.public.seed@seed__int8__float8__date_idx AS tab_1335, defaultdb.public.seed@seed__int8__float8__date_idx AS tab_1336, defaultdb.public.seed AS tab_1337 LIMIT 17;
        END IF;
        NULL;
        EXIT WHEN EXISTS (WITH with_276 (col_2168) AS (SELECT * FROM (VALUES (1724605171)) AS tab_1338 (col_2168)), with_277 (col_2169) AS (SELECT * FROM (VALUES ('-75 years -11 mons -251 days -17:18:33.949733'), ('-71 years -7 mons -1 days -00:33:41.408845'), ('31 years 130 days 20:29:33.515078'), ('00:00:00'), ('-72 years -1 mons -202 days -19:47:29.032149'), ('-4 mons -760 days -14:20:56.033043')) AS tab_1339 (col_2169)), with_278 (col_2170) AS (SELECT * FROM (VALUES ('B'), ('')) AS tab_1340 (col_2170)) SELECT '[-1.8953724,-0.4354469,0.33804223,1.0959439,-1.6515361,0.07848346,-0.60566723,1.5132728,0.8167821,0.5369794,1.1160319,1.7494828,0.42835915,-1.4563392,-2.9630651,1.2361355,-0.87962264,2.030708,-0.28000024,1.0887539,1.482931,-0.67387104,-2.1597614,0.5438532,0.057551995,-0.15394673,0.4177094,0.19525738,1.3639203,-1.6270634,-1.2892803,-0.6431209,-1.2991941,-1.7226549,0.8086041,-0.65116686,1.4330951,-0.0583802,0.29851577,0.41827077,0.056676205,0.69941247,0.8481294,-0.089288756,-0.3805584,0.25824153,2.2863288,0.5426884,1.8294307,0.17304501,-0.11690498,0.20198955,-0.6087622,0.0848198,0.9849374,0.36980176,1.4007933,0.7569631,-0.5760042,-0.89219993,-1.632751,0.11817601,1.1395646,0.6772345,-1.0935339,0.3029954,-0.9061749,-2.3961189,-1.7730423,0.31278828,1.1338928,0.20402779,0.6697234,-1.2215759,-0.72592205,1.6576512,-0.48144382,1.1840115,1.0562338,-1.0653292,-0.03357926,1.3588232,-1.3301226,0.64725167,0.17837515,-0.3699021,-0.64169705,-0.82872075,0.32989487,0.110170834,1.3241515,-0.30493835,-2.150736,0.49440834,0.48028907,-0.123921044,0.3005524,-0.43623495,0.8986038,-0.41064426,-1.0812403,1.6500946,-0.44782847,-0.11135803,1.2138631,-0.039451,0.106775776,-1.406416,0.9109457,1.665857,-1.5599102,-0.31468797,-0.6583459,-0.56599367,2.0539663,0.6092671,0.8223509,0.6976953,0.034386285,0.43693593,-1.3107013,-0.76780933,1.7622321,0.2808852,0.13697183,-1.53476,-1.708858,1.0965705,0.671808,0.07395631,-1.2308108,1.5428793,-0.3017068,0.59092224,-0.01872497,0.41320932,-0.4971843,-1.1906137,-0.2822678,-2.0925536,-0.6993071,-0.48367527,1.1716704,0.7412778,-0.49848223,-1.112755,-3.0775838,-0.14335966,0.86597013,-1.9310223,0.554609,0.07379981,-0.13983093,0.76245666,-1.0071093,0.16216248,2.5729964,-2.3143978,0.2868827,0.357073,-0.17223567,-0.3617854,0.28636467,-1.5387723,0.16040502,-0.9573212,-0.8121023,-0.51514894,1.5294997,2.0246787,1.7659197,-0.5246781,-1.8639603,1.1942502,0.65066075,1.2352191,-0.45981592,-1.153261,0.6794819,1.7029774,-0.26814288,0.38250628,-1.6697539,0.26006976,-0.59407663,-0.25080514,0.45847073,-1.475407,0.45036805,0.119456485,0.769673,0.4611636,-1.1028584,0.94585645,-0.5395548,-0.90221673,2.1731179,1.7284373,0.38175288,-0.6701439,0.9123127,0.2682758,-0.50790507,-1.674945,-0.09031421,-1.1475996,-0.3202392,-0.22575235,0.57444537,-0.2950753,-1.4166139,0.49054515,-0.31174007,0.94190025,-2.1646402,-0.9634423,1.580813,-1.0830083,0.11066873,0.00089755445,1.1744533,-1.8919019,-0.4760428,-0.95045936,-0.9566621,0.28673443,0.89006793,-1.4876397,-0.06604667,-0.65216076,0.13013372,0.33611527,-0.047419704,0.68410283,-0.6376756,-2.4389312,0.80618036,-0.12332607,-0.5239293,-0.53525794,0.54962003,-1.5788618,-0.41413966,-0.7494067,1.3847406,-2.0358713,0.58251095,0.55923134,1.5584286,0.2542855,0.0733182,1.0419198,1.7482917,1.5960783,-0.62639904,1.5885196,-1.6932162,-0.8182709,-0.6990818,0.629698,-1.4337467,1.5803747,-1.5269052,1.536877,-2.0578153,0.4384691,0.8033,1.5544254,0.0012137765,-0.34185588,-0.99009323,-1.1329188,0.73557496,-0.96608645,1.2935268,-0.15599763,-2.6928818,1.0746992,-1.2373525,0.4597044,-1.07624,-0.30074203,0.12852325,0.9679984,0.73398024,-0.68512934,-0.4182882,1.3792323,1.3324844,-0.008094284,0.03897533,-0.9897224,-2.163168,-1.6409582,-0.99513817,0.19851011,-0.3023706,-0.1209241,0.8755053,-0.29064292,-1.2477179,-0.16927065,1.5754182,0.4001945,-0.103873484,-0.6024941,0.67577964,0.025583312,-0.5749099,0.3145261,-0.17630203,0.7979027,-0.9068193,1.8692088,0.098962314,-0.5110081,0.40764055,1.1804377,-0.10758868,-0.42601153,-0.96251315,-0.25898668,-0.08210036,-1.055891,-0.023976985,0.11267313,0.06860851,-1.3862164,0.94758415,-1.2400384,0.5142011,0.2639373,-0.67872036,0.038001385,0.31391972,1.8557042,0.2985821,-0.059608508,0.16935213,0.3439718,0.25517026,0.91770786,1.1787336,-0.16551161,-1.0534465,0.1853052,-0.6443088,0.03098441,-1.6188613,1.2119251,-0.4432927,0.8595806,1.008084,-0.66636544,-0.82832557,-0.6388049,0.97138286,0.071644366,0.550023,0.5086342,-0.37525654,0.012984951,1.5102277,0.28031227,-0.97449756,-0.8684792,-0.15847714,0.14951198,-0.43447632,0.9112088,-0.5173345,0.19135219,0.01175719,-0.18224424,-1.0471282,-0.6464,-0.7153109,-1.7308521,0.57573724,1.2093915,-0.4729985,0.9238809,-0.44823822,0.06809785,-0.65210855,-0.49101335,-0.67291915,0.42162794,-0.5965021,-0.39429104,0.5839267,1.0952649,-0.09288994,-0.56955034,0.68815416,-0.62549055,-0.63418156,-1.0682063,-0.82714844,0.09684773,0.819451,0.09210444,-0.46011695,0.09763695,0.7261929,0.54387903,1.561483,-0.49554762,0.3230893,-3.23873,0.905301,-0.997558,-0.3895355,-0.39662984,0.99218464,0.80714124,2.289605,1.4462594,0.5639643,0.62697256,1.5758765,0.38839316,-2.8821378,0.9393881,0.44589037,-0.25910044,-0.4659664,0.94394237,1.0891583,-0.9074785,0.2123801,-0.23622768,-1.5433551,0.82062894,0.47468328,-0.5779589,-0.8064127,-0.89375836,0.4107937,0.55337006,1.7989396,-0.1680741,-1.0068969,-0.16496494,0.4625207,-0.860535,0.8558721,-0.39438128,0.40503827,0.09180121,-0.02642961,-0.33771858,0.13867141,-0.65976816,0.44133422,-0.03351468,0.08120018,0.865229,0.2273199,-1.1284261,-1.5710878,-0.16062778,-0.5448919,0.2951326,-1.738571,-0.7140519,-0.11468323,0.7857934,1.1847267,-1.1328279,-0.09785601,-1.2655581,0.41369477,1.2595316,1.4925491,-0.50858057,1.7968984,0.43911767,-0.54761845,1.7774194,0.38038632,-0.7641977,1.0179073,-0.38633078,1.198234,-1.3069257,-1.9071494,-0.49315915,1.8113933,0.09870178,-1.3093884,-0.7192045,1.4696643,-0.038726013,1.1650944,-0.9228386,0.9612254,0.09672245,-0.22036023,1.1422305,-0.08548195,2.3477397,0.6727637,-0.6820685,-0.6082859,1.3710942,-0.92684925,-1.3091067,-1.0239674,0.4180606,-1.2052224,-0.6814359,0.30834302,-0.16989689,-0.18625315,-2.6063914,-1.491636,-0.41938385,0.8803421,-0.5855332,-0.5533824,0.4435652,0.44045708,-1.3939141,2.0704544,0.48276678,0.49209195,1.830178,-0.022099929,-0.4595976,-0.66984695,-0.3175871,-0.95878094,0.8698407,2.304643,1.1270833,0.7623989,0.46387908,-0.2598671,0.25759235,0.2869584,1.5307817,-2.343673,-0.14281921,0.4644549,-0.08853788,0.80866367,-0.83422214,-0.8999482,-0.54945874,0.717178,0.41174066,0.06030508,1.663294,-1.799499,0.34831944,0.90781844,0.98167914,1.4895761,-1.5496064,-0.13572213,0.7483348,-1.1921532,-1.1600937,0.40761697,-0.99199283,-1.822955,0.24509485,-0.108497806,0.40136766,0.618716,0.5625601,0.114245586,-0.6087587,0.76303476,1.0241239,0.09074174,2.086524,1.4101894,-1.1500733,-0.3320713,-0.1008438,-1.3624473,-1.4723452,-1.5699999,-1.1861116,-0.9798255,0.67139494,0.9596749,-0.80223805,-0.8687434,-0.9222237,-2.0565073,1.3183582,-0.382338,0.6628444,1.4331213,-0.49472967,1.4554625,1.0812843,-0.35793248,-0.687481,2.2787905,1.0724537,-0.9255714,-0.6414349,0.21195507,0.9972359,-0.94612366,-1.9930255,-0.6030083,0.63449556,0.8468272,0.68509346,0.50300545,-1.4555055,0.7454272,0.7844174,-1.7435757,1.0948232,-0.11662971,-0.6627609,-0.23606949,0.3620281,-2.0573168,0.575855,-0.9673223,1.3172156,-0.13621369,-0.40385684,-0.49674892,-0.9046938,0.19525619,0.87736344,0.063035525,-0.40551296,-0.745598,1.1497364,-2.3416004,-1.6565846,-0.05986504,-0.66459054,-2.1863604,1.684426,0.123124756,1.1281654,0.08795573,0.5244254,0.04127385,-1.2393359,0.3299313,0.26864627,-1.6913035,0.8155854,-0.38955534,-1.493814,-1.2158548,-0.3456734,-0.7979554,1.7914466,-0.9550593,-0.86035967,-0.27965674,2.2909594,-0.19523752,-1.6602545,2.6310873,0.20224093,-1.2483634,0.78263533,1.1960708,1.8889924,-0.20812312,0.08910251,-0.34879336,-0.42185906,0.9251079,-1.3387817,0.24102756,1.8140991,0.10002111,-0.31457716,-0.07408105,0.3716479,-0.40157765,-0.017874278,-0.90506244,0.017779382,-0.55564487,1.2386858,-1.5269033,-1.1195824,0.76267374,0.28723654,-0.5678135,-0.1364865,-0.94096696,1.2569462,-0.08108643,0.6425783,-1.5243359,-1.4193217,-0.048449103,-0.5067812,-1.0271671,-0.011602508,0.5747331,0.4741436,-0.23732065,-0.6135852,-1.1628025,0.5436029,-0.082669124,0.08740234,0.2939035,-0.09727116,1.30741,1.0354666,1.18467,0.8710088,-0.6214939,0.5345046,-0.4259226,0.6365181,0.7704157,0.7564135,-0.40319678,0.37700897,0.4464361,-0.13360439,-0.03409716,1.536735,1.243011,-1.3368787,-1.1123022,-1.1063181,-0.050957657,-1.2128466,-1.3669655,-0.39252776,2.416964,-0.9784335,0.0941287,-0.38748184,0.5288165,-0.17528148,0.16496986,-1.5703392,1.3433133,0.47961918,-0.6768596,1.5650606,1.8788943,0.5640791,-1.2893574,-0.7493853,-0.9221292,-0.16665122,-1.0198822,-0.30025917,-0.032355912,-0.8886719,2.1100981,-0.6026695,1.324099,0.5663427,-0.042000607,-0.7333406,0.435478,1.3081203,-0.400634,0.37757242,0.7715887,1.5700754,-0.24900743,0.66588867,-0.25548586,-0.4098286,0.3429166,-0.15384026,-0.73406154,0.08617706,-0.458666,1.0714215,0.34683353,0.60480815,-0.77681166,-0.3894418,0.6854147,0.13619442,1.0708063,-0.2798807,0.095580444,-0.26909205,0.2589318,0.71659017,-1.1386096,-0.00019784643,-0.062174186,0.2999169,-1.704283,-0.23992707,1.5368394,0.14935309,1.3747958,0.14293876,0.8964437,-2.1764748,0.5447451,-0.24914588,0.3989797,-0.74190795,-0.7616715,-0.21250677,-0.40197268,0.09838826,0.36974552,0.14601542,-1.3351924,0.78579897,-0.9795696,-0.521212,0.27146292,0.7938306,-0.55292225,0.44714236,-0.08346681,0.9289059,0.046848733,-0.9712301,-1.142066,-0.8464405,1.093555,1.7861656,-0.53778535,-0.2910865,-1.6937759,0.4002006,-0.46192777,0.57597136,0.9108149,1.8755744,-0.3915267,0.63994956,-0.273325,0.45715192,0.22051983,0.36357978,0.5701256,-1.3994237,0.9470288,-0.5424757,0.96042746,-1.2835046,-0.88792187,-0.46471077,-1.6726978,-0.9382151,-1.453589,0.37794945,-0.94398326,-0.6536227,-1.8385283,1.4783729,1.4299002,-0.30825174,-1.169563,-0.70007724,-0.9323018,0.48210216,0.75581044,1.1665695,-0.43368825,0.5506862,0.18296956,1.1433557,-1.4282411,1.2585359,2.7453125,-0.06570475,-0.5362388,0.5760894,-0.072394215,0.16376513,0.53520274,0.7524695,-1.7744282,-0.009802261,-0.031405415,-0.9101092,0.40183187,1.2699785,-0.9423662,-0.8750913]' AS col_2171 FROM defaultdb.public.seed AS tab_1341 ORDER BY tab_1341._int8 NULLS LAST);
        END LOOP;
        WHILE true LOOP
        NULL;
        EXIT WHEN true;
        CONTINUE WHEN false;
        END LOOP;
        SELECT NULL AS col_2172, tab_1342._date AS col_2173, (3.84431280881181702E+36::DECIMAL ^ NULL::INT8)::DECIMAL AS col_2174, e'"s\'NBG ' AS col_2175, false AS col_2176, tab_1342.col_973 AS col_2177, ('[{"a": "JoK0zx1", "baz": 1.9692652492978857}, [[0.23285452620536118, 0.027608006380234196], true, null, true], {"JXzjdlNQ": null}, "baz"]'::JSONB->e'"4xR1|}\n'::STRING)::JSONB AS col_2178, (SELECT tab_1342._interval AS col_2179 LIMIT 1) AS col_2180, -0.7159425020217896 AS col_2181, cbrt(tab_1342.crdb_internal_mvcc_timestamp::DECIMAL)::DECIMAL AS col_2182, tab_1342.crdb_internal_origin_timestamp AS col_2183, NULL AS col_2184, 1.8901202942751993 AS col_2185 FROM defaultdb.public.seed@seed__int8__float8__date_idx AS tab_1342 LIMIT 99;
        END;
        $$;
--- FAIL: TestExplainGist/main (69.44s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=2</code>
 - <code>shard=2</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExplainGist.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-58755

--- a/pkg/ccl/testccl/sqlccl/explain_test.go
+++ b/pkg/ccl/testccl/sqlccl/explain_test.go
@@ -203,7 +203,6 @@ func TestExplainGist(t *testing.T) {
 					"type check failed while initializing stat",                          // #125620
 					"argument expression has type RECORD, need type USER DEFINED RECORD", // #139910
 					"invalid datum type given: RECORD, expected RECORD",                  // #140773
-					"not in index", // #148405
 				} {
 					if strings.Contains(err.Error(), knownErr) {
 						// Don't fail the test on a set of known errors.
@@ -305,11 +304,11 @@ func TestExplainGist(t *testing.T) {
 				} else {
 					logStmt(stmt, true /* successful */)
 				}
-			case <-time.After(time.Minute):
+			case <-time.After(3 * time.Minute):
 				t.Log(stmts.String())
 				sl := allstacks.Get()
 				t.Logf("stacks:\n\n%s", sl)
-				t.Fatalf("stmt wasn't canceled by statement_timeout of 0.1s - ran at least for 1m: %s", stmt)
+				t.Fatalf("stmt wasn't canceled by statement_timeout of 0.1s - ran at least for 3m: %s", stmt)
 			}
 		}
 	})

